kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/AdminResource.scala

```diff
@@ -64,6 +64,26 @@ private[v1] class AdminResource extends ApiRequestContext with Logging {
     Response.ok(s"Refresh the hadoop conf for ${fe.connectionUrl} successfully.").build()
   }
 
+  @ApiResponse(
+    responseCode = "200",
+    content = Array(new Content(
+      mediaType = MediaType.APPLICATION_JSON)),
+    description = "refresh the user defaults configs")
+  @POST
+  @Path("refresh/user_defaults_conf")
+  def refreshUserDefaultsConf(): Response = {
```

Comment:
shall we lock this method ?

Reply:
`refreshUserDefaultsConf` of KyuubiServer is synchronized on kyuubiServer.conf to prevent concurrent racing.